views/home-sidebar.pug

```diff
@@ -2,10 +2,10 @@ h4.sidebar-item#sidebar-name #{user.fname} #{user.lname}
 h3.sidebar-item#sidebar-points #{user.points} Points
 p.sidebar-item Questions attempted: #{user.attemptedCount}
 p.sidebar-item Questions successfully answered: #{user.answeredCount}
-if user.attempted == 0
+if (user.answeredCount + user.attemptedCount) === 0
   p.sidebar-item You have not yet attempted any questions.
 else
-  p.sidebar-item You have a #{((user.answeredCount / user.attemptedCount) * 100).toFixed(2)}% success rate.
+  p.sidebar-item You have a #{((user.answeredCount / (user.answeredCount + user.attemptedCount)) * 100).toFixed(2)}% success rate.
```

Comment:
Again, I think the attemptedCount includes the answeredCount. Please feel free to let me know if I'm wrong :)

Reply:
I tried to follow this:
Students attempted the question, means they failed to answer correctly 
Students answered the question, means they answered it correctly